extensions/interactions/LogicProof/LogicProof.js

```diff
@@ -49,22 +49,38 @@ oppia.directive('oppiaInteractiveLogicProof', [
         );
         $scope.questionData.language.operators = $scope.typing[0].operators;
 
-        $scope.assumptionsString = logicProofShared.displayExpressionArray(
-          $scope.questionData.assumptions,
-          $scope.questionData.language.operators);
-        $scope.targetString = logicProofShared.displayExpression(
-          $scope.questionData.results[0],
-          $scope.questionData.language.operators);
+        $scope.displayExpression = function(exp) {
```

Comment:
These are just wrapper functions to make the following code a bit more concise so they don't merit comments. If you really don't like them then I can get rid of them.

Done.